frontend/style.scss

```diff
@@ -513,6 +513,10 @@ button + button {
   color: $light-grey-1;
 }
 
+.redploy-button{
+  margin-left: 330px;
```

Comment:
This push from the left could break the responsive layout. you may want to `float: right;` instead or add a media query that resets margin-left here.

```
.redploy-button {
  margin-left: 330px;
  @media only screen and (max-width: 768px) {
    margin-left: 0;
  }
}
```

Reply:
hey @dancrew32 I tried float:right, it would make it a little lower, and I tried the responsiveness of this code on browser in different views and looked fine. But I will give it another try, that you mention.